### PR TITLE
Make the `lib64` symlink relative

### LINKED ISSUE #35: Use a relative symlink for `lib64`
It turns out to be on purpose: https://github.com/python/cpython/commit/581c29f8fe20a6d95f036a3499b55cc2c587b25f

### PR DESCRIPTION
**Pull request by ur4t**
Fixes #35

### CODE REVIEW

**Comment by brettcannon** [comment]
Can you explain why you consider the symlink "wrong"? I'm assuming because it's an absolute path, but I don't quite see how that's inaccurate.

**Comment by brettcannon** [comment]
I found https://github.com/python/cpython/commit/581c29f8fe20a6d95f036a3499b55cc2c587b25f which documents that it's on purpose.

**Review by brettcannon — CHANGES_REQUESTED** [review]
Please add a test using `pathlib.Path.readlink()` to verify that the symlink is relative.

**Comment by ur4t** [comment]
> Can you explain why you consider the symlink "wrong"? I'm assuming because it's an absolute path, but I don't quite see how that's inaccurate.

It is an absolute path in test cases, which is provided by `tmp_path_factory.mktemp()`. But we did not make `env_path` an absolute path when we simply run `python3 microvenv.py` (or with other relative path names).

I found this bug via running `python3 microvenv.py && readlink -f .venv/lib64`, which was failed because `.venv/lib64` was symlinked to a relative path `.venv/lib` then.

So I chose the title "fix improper symlink".

> I found [python/cpython@581c29f](https://github.com/python/cpython/commit/581c29f8fe20a6d95f036a3499b55cc2c587b25f) which documents that it's on purpose.

Great!

> Please add a test using `pathlib.Path.readlink()` to verify that the symlink is relative.

Added.

File: test_microvenv.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ def test_lib64(full_venv, micro_venv):
         assert micro_lib64.exists()
         assert micro_lib64.is_symlink()
         assert micro_lib64.resolve() == (micro_venv / "lib")
+        assert pathlib.Path.readlink(micro_lib64).is_relative_to("lib")
```
Comment (brettcannon) [inline test_microvenv.py:82]: `is_relative_to()` is a bit too flexible to make sure this doesn't change, so I would prefer to be a bit strict about exactly what the symlink is set to.

```suggestion
        assert os.fsdecode(pathlib.Path.readlink(micro_lib64)) == "lib"
```

**Review by brettcannon — CHANGES_REQUESTED** [review]

File: test_microvenv.py [hunk 2]
```diff
@@ -79,6 +79,7 @@ def test_lib64(full_venv, micro_venv):
         assert micro_lib64.exists()
         assert micro_lib64.is_symlink()
         assert micro_lib64.resolve() == (micro_venv / "lib")
+        assert os.fsdecode(pathlib.Path.readlink(micro_lib64)) == "lib"
```
Comment (brettcannon) [inline test_microvenv.py:82]: `pathlib.Path.readlink()` was added in Python 3.9.

```suggestion
        # pathlib.Path.readlink() was added in Python 3.9.
        assert os.readlink(micro_lib64) == "lib"
```

**Review by brettcannon — CHANGES_REQUESTED** [review]

**Comment by brettcannon** [comment]
Thanks for the PR!

**Pull request merged**